README.md

```diff
@@ -2,4 +2,84 @@
 [![codecov](https://codecov.io/gh/mendersoftware/artifacts/branch/master/graph/badge.svg)](https://codecov.io/gh/mendersoftware/artifacts)
 [![Go Report Card](https://goreportcard.com/badge/github.com/mendersoftware/artifacts)](https://goreportcard.com/report/github.com/mendersoftware/artifacts)
 
-# artifacts
+Mender Artifacts Library
+==============================================
+
+Mender is an open source over-the-air (OTA) software updater for embedded Linux
+devices. Mender comprises a client running at the embedded device, as well as
+a server that manages deployments across many devices.
+
+This repository contains the artifacts library, which is used by the
+Mender client, command line interface, server and for build integration with the Yocto Project. 
+
+The artifacts library makes it easy to programmatically work with a Mender artifact, which
+is a file that can be recognized by its `.mender` suffix. Mender artifacts
+can contain binaries, metadata, checksums, signatures and scripts that are
+used during a deployment. The artifact format acts as a wrapper, and
+uses the `tar` format to bundle several files into one.
+
+In its simplest form, an artifact contains just a rootfs image,
+along with its checksum, id and device type compatibility.
+
+
+The artifacts library might also be useful for other updaters or
+purposes. We are always happy to see other uses of it!
+
+
+![Mender logo](https://mender.io/user/pages/04.resources/_logos/logoS.png)
+
+
+## Getting started
+
+To start using Mender, we recommend that you begin with the Getting started
+section in [the Mender documentation](https://docs.mender.io/).
+
+
+## Using the library
+
+To build the artifacts library, you can follow these steps:
+
+```
+git clone https://github.com/mendersoftware/artifacts.git
+cd artifacts
+go build
```

Comment:
ok, that didn't work for me (probably missing some PATH magic).
I remove this part as it seems to be pretty standard go stuff.